### ISSUE TITLE
Colored icons?

### ISSUE THREAD

**Issue by iguanajuice** [iguanajuice/sway-font-awesome#4]
Opening this issue to see if anyone would be interested in colored icons.

Obviously, they will only be single color, but I think it could help with at-a-glance legibility nonetheless.

Thoughts?

**Comment by tkna91** [comment]
While it may be less distracting to go in a minimalistic direction with no colors, it may be more convenient to have colors for easy identification.
It might be a good idea to create a replacement script or include an example command in the README description so that users can generate a colorless version as well.

A version with colors is implemented here, but it is not very simple as it is made by rust and requires daemon activation.

https://github.com/allie-wake-up/swaycons

I feel that your method is more convenient as all I have to do is to include the configuration file.

**Comment by tkna91** [comment]
I guess the main advantage of the above is that it can dynamically respond to changes in window titles, etc. I think this is especially useful if you want to manage windows in sway tabs with Firefox as SDI.
I don't use it that way, so I prefer to include config.

**Comment by iguanajuice** [comment]
> It might be a good idea to create a replacement script or include an example command in the README description so that users can generate a colorless version as well.

I was thinking of having two different files in the repo, `icons` and `icons-color`.

**Comment by tkna91** [comment]
It would be nice if it could be automated with github-action or something.

**Comment by tkna91** [comment]
No, it seems to me that a shell script or other method such as generating it before git commit, etc. would be better and less dependent on the vendor.
